mofa/simulation/lammps.py

```diff
@@ -8,9 +14,103 @@ class LAMMPSRunner:
     Args:
         lammps_command: Command used to launch LAMMPS
     """
+    lammps_command = "npt_tri"
+    lmp_sims_root_path = "lmp_sims"
+    cif_files_root_path = "cif_files"
+    cif_files_paths = []
 
-    def __init__(self, lammps_command: str):
-        self.lammps_command: str = lammps_command
+    def __init__(self, lammps_command: str = "npt_tri", lmp_sims_root_path: str = "lmp_sims", cif_files_root_path: str = "cif_files"):
+        """Read cif files from input directory, make directory for lammps simulation input files
+
+        Args:
```

Comment:
Are these input arguments different for MOF being simulated? The arguments for the LAMMPSRunner class should be used across all runs of a each MOF (e.g., all will use the same path to LAMMPS). Arguments for a specific MOF (e.g., the input CIF) should be in the function calls.

Reply:
FWIW, it looks like "cif_files" is not used. Could we just delete it?